.github/workflows/ruby.yml

```diff
@@ -0,0 +1,59 @@
+name: Ruby
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test_and_coverage:
+
+    runs-on: ubuntu-latest
+    name: Ruby Test + Coverage
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 3.0
+        bundler-cache: true # Runs bundle install and caches gems. See the ruby_test.yml
+                            # example if you need more control over bundler.
```

Comment:
This `ruby_test.yml` refers to the documentation in your repo, right? Either remove the comment or be more specific about where to find it?